composer.json

```
@@ -17,10 +17,10 @@
         }
     },
     "require": {
-        "php" : ">=7.2"
+        "php" : "8.*"
```

Par cohérence avec le retour de nico sur la lib pussle/amqp
```suggestion
        "php" : ">=8.0"
```

---

b185d5ec8ee3b286a5f1a69f20c65c4a03a4126f